tests/contrib/test_pos_classifier.py

```diff
@@ -0,0 +1,49 @@
+from galahad.server.contrib.pos.spacy_pos import SpacyPosClassifier
+from galahad.server.dataclasses import Document
+
+
+def test_spacy_pos():
+    clf = SpacyPosClassifier("en_core_web_sm")
+
+    text = "I am jealous. Peter received such a beautifully crafted gift."
+
+    annotations = {
```

Comment:
If you write a test like this, then you run into issues when the model changes. I would rather check that all tokens have a label here and then in another test, that your predictions have a reasonable performance on a simple corpus like in sentence_classifier tests